sdk/src/main/java/com/kin/ecosystem/Kin.java

```diff
@@ -266,6 +266,31 @@ public void onFailure(KinEcosystemException exception) {
 		}
 	}
 
+	public static MigrationManager createMigrationManager(Context context, @NonNull String appId) {
+		KinEnvironment kinEnvironment = ConfigurationImpl.getInstance().getEnvironment();
+
+		final String oldNetworkUrl = kinEnvironment.getOldBlockchainNetworkUrl();
+		final String oldNetworkId = kinEnvironment.getOldBlockchainPassphrase();
+
+		final String newNetworkUrl = kinEnvironment.getNewBlockchainNetworkUrl();
+		final String newNetworkId = kinEnvironment.getNewBlockchainPassphrase();
+
+		final String issuer = kinEnvironment.getOldBlockchainIssuer();
+		final String migrationServiceUrl = kinEnvironment.getMigrationServiceUrl();
+
+		MigrationNetworkInfo migrationNetworkInfo = new MigrationNetworkInfo(oldNetworkUrl, oldNetworkId,
+				newNetworkUrl, newNetworkId, issuer, migrationServiceUrl);
+
+		final BlockchainSource.Local local = BlockchainSourceLocal.getInstance(context);
+		final BlockchainSource.Remote remote = BlockchainSourceRemote
+				.getInstance(getInstance(context).executorsUtil, eventLogger);
+		MigrationManager migrationManager = new MigrationManager(context, appId, migrationNetworkInfo,
+				new KinBlockchainVersionProvider(local, remote),
+				new MigrationEventsListener(EventLoggerImpl.getInstance()), KIN_ECOSYSTEM_STORE_PREFIX_KEY);
+		migrationManager.enableLogs(Logger.isEnabled());
+		return migrationManager;
+	}
+
```

Comment:
why is that changed?

Reply:
createMigrationManager is used in my class needed to access it

Reply:
it can't be public this is the interface for the DS.
need to think of another solution.

Reply:
@OrenZak can i put this code as  private function in my class